### PR TITLE
Adding new Pearlboards boards to directory

### PR DESCRIPTION
**Pull request by Koobaczech**
## Description

Added 4 new boards to the pearlboards directory. Using a clean branch off a fresh clone.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x ] I have updated the documentation accordingly.
- [x ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x ] I have added tests to cover my changes.
- [ x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/pearlboards/atlas/keymaps/all/keymap.c [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    /* First encoder */
+    if (index == 0) {
+        if (clockwise) {
+            tap_code(KC_AUDIO_VOL_UP);
+        } else {
+            tap_code(KC_AUDIO_VOL_DOWN);
+        }
+    }
+    return true;
+}
```
Comment (drashna) [inline keyboards/pearlboards/atlas/keymaps/all/keymap.c:47]:
```suggestion
```

File: keyboards/pearlboards/atlas/keymaps/via/keymap.c [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    /* First encoder */
+    if (index == 0) {
+        if (clockwise) {
+            tap_code(KC_AUDIO_VOL_UP);
+        } else {
+            tap_code(KC_AUDIO_VOL_DOWN);
+        }
+    }
+    return true;
+}
```
Comment (drashna) [inline keyboards/pearlboards/atlas/keymaps/via/keymap.c:61]:
```suggestion
```

File: keyboards/pearlboards/atlas/atlas.c [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+ #include "atlas.h"
```
Comment (drashna) [inline keyboards/pearlboards/atlas/atlas.c:17]: This allows for a default behavior to be defined, without the need for the user to specify the behavior.  This is very useful for the configurator which doesn't have a way to configure this behavior, currently.

```suggestion
#include "atlas.h"

#ifdef ENCODER_ENABLE
bool encoder_update_kb(uint8_t index, bool clockwise) {
    if (!encoder_update_user(index, clockwise)) { return false; }
    if (index == 0) {
        if (clockwise) {
            tap_code_delay(KC_VOLU, 10);
        } else {
            tap_code_delay(KC_VOLD, 10);
        }
    }
    return true;
}
#endif
```

File: keyboards/pearlboards/atlas/rules.mk [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes        # Enable keyboard RGB underglow
+AUDIO_ENABLE = yes           # Audio output
+DIP_SWITCH_ENABLE = no
+ENCODER_ENABLE = yes
+OLED_DRIVER_ENABLE = no      # Future release
+LTO_ENABLE = yes
+HAPTIC_ENABLE += DRV2605L
```
Comment (drashna) [inline keyboards/pearlboards/atlas/rules.mk:26]:
```suggestion
ENCODER_ENABLE = yes
HAPTIC_ENABLE = yes
HAPTIC_DRIVER = DRV2605L

LTO_ENABLE = yes
```

File: keyboards/pearlboards/pearl/config.h [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+#define FB_LOOPGAIN 1 // For  Low:0, Medium:1, High:2, Very High:3
+/* Motor settings */
+#define RATED_VOLTAGE 3
+#define V_PEAK 5
+
+/*== all animations enabled ==*/
+/* If RGBLIGHT_SLEEP defined, the RGB lighting will be switched off when the host goes to sleep */
+#define RGB_DI_PIN B3
+#ifdef RGB_DI_PIN
+#    define RGBLIGHT_ANIMATIONS
```
Comment (drashna) [inline keyboards/pearlboards/pearl/config.h:65]: Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

File: keyboards/pearlboards/pearl/rules.mk [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes        # Enable keyboard RGB underglow
+AUDIO_ENABLE = yes           # Audio output
+DIP_SWITCH_ENABLE = no
+ENCODER_ENABLE = no
+OLED_DRIVER_ENABLE = no      # Future release
+LTO_ENABLE = yes
+HAPTIC_ENABLE += DRV2605L
```
Comment (drashna) [inline keyboards/pearlboards/pearl/rules.mk:26]:
```suggestion
LTO_ENABLE = yes
HAPTIC_ENABLE  = yes
HAPTIC_DRIVER = DRV2605L
```

File: keyboards/pearlboards/zeus/config.h [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+#define FB_LOOPGAIN 3 // For  Low:0, Medium:1, High:2, Very High:3
+/* Motor settings */
+#define RATED_VOLTAGE 2
+#define V_PEAK 5
+
+/*== all animations enabled ==*/
+/* If RGBLIGHT_SLEEP defined, the RGB lighting will be switched off when the host goes to sleep */
+#define RGB_DI_PIN D5
+#ifdef RGB_DI_PIN
+#    define RGBLIGHT_ANIMATIONS
```
Comment (drashna) [inline keyboards/pearlboards/zeus/config.h:76]: Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
// #define RGBLIGHT_EFFECT_RGB_TEST
// #define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

File: keyboards/pearlboards/zeus/keymaps/all/keymap.c [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+    /* First encoder */
+    if (index == 0) {
+        if (clockwise) {
+            tap_code(KC_AUDIO_VOL_UP);
+        } else {
+            tap_code(KC_AUDIO_VOL_DOWN);
+        }
+    }
+    return true;
+}
```
Comment (drashna) [inline keyboards/pearlboards/zeus/keymaps/all/keymap.c:49]:
```suggestion
```

File: keyboards/pearlboards/zeus/keymaps/via/keymap.c [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+    /* First encoder */
+    if (index == 0) {
+        if (clockwise) {
+            tap_code(KC_AUDIO_VOL_UP);
+        } else {
+            tap_code(KC_AUDIO_VOL_DOWN);
+        }
+    }
+    return true;
+}
```
Comment (drashna) [inline keyboards/pearlboards/zeus/keymaps/via/keymap.c:65]:
```suggestion
```

File: keyboards/pearlboards/zeus/keymaps/all/keymap.c [hunk 10]
```diff
@@ -0,0 +1,49 @@
+/*
```
Comment (drashna) [inline keyboards/pearlboards/zeus/keymaps/all/keymap.c:1]: please rename this folder to "default"

File: keyboards/pearlboards/zeus/rules.mk [hunk 11]
```diff
@@ -0,0 +1,26 @@
...
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes        # Enable keyboard RGB underglow
+AUDIO_ENABLE = yes           # Audio output
+DIP_SWITCH_ENABLE = no
+ENCODER_ENABLE = yes
+OLED_DRIVER_ENABLE = no      # Future release
+LTO_ENABLE = yes
+HAPTIC_ENABLE += DRV2605L
```
Comment (drashna) [inline keyboards/pearlboards/zeus/rules.mk:26]:
```suggestion
ENCODER_ENABLE = yes
LTO_ENABLE = yes
HAPTIC_ENABLE = yes
HAPTIC_DRIVER = DRV2605L
```

File: keyboards/pearlboards/zeuspad/keymaps/all/keymap.c [hunk 12]
```diff
@@ -0,0 +1,86 @@
+/*
```
Comment (drashna) [inline keyboards/pearlboards/zeuspad/keymaps/all/keymap.c:1]: please rename this to "default"

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/pearlboards/atlas/config.h [hunk 13]
```diff
@@ -0,0 +1,92 @@
...
+#define FB_LOOPGAIN 3 // For  Low:0, Medium:1, High:2, Very High:3
+/* Motor settings */
+#define RATED_VOLTAGE 3
+#define V_PEAK 5
+
+/*== all animations enabled ==*/
+/* If RGBLIGHT_SLEEP defined, the RGB lighting will be switched off when the host goes to sleep */
+#define RGB_DI_PIN B6
+#ifdef RGB_DI_PIN
+#    define RGBLIGHT_ANIMATIONS
```
[the remark this answers is not in the document]
Comment (Koobaczech) [reply keyboards/pearlboards/atlas/config.h]: Done sir

File: keyboards/pearlboards/atlas/atlas.c (hunk 3 above)
Comment (Koobaczech) [reply keyboards/pearlboards/atlas/atlas.c]: Done sir

File: keyboards/pearlboards/atlas/rules.mk (hunk 4 above)
Comment (Koobaczech) [reply keyboards/pearlboards/atlas/rules.mk]: Done sir

File: keyboards/pearlboards/pearl/config.h (hunk 5 above)
Comment (Koobaczech) [reply keyboards/pearlboards/pearl/config.h]: Done sir

File: keyboards/pearlboards/pearl/rules.mk (hunk 6 above)
Comment (Koobaczech) [reply keyboards/pearlboards/pearl/rules.mk]: Done sir

File: keyboards/pearlboards/zeus/keymaps/all/keymap.c (hunk 8 above)
Comment (Koobaczech) [reply keyboards/pearlboards/zeus/keymaps/all/keymap.c]: Done sir

File: keyboards/pearlboards/zeus/keymaps/via/keymap.c (hunk 9 above)
Comment (Koobaczech) [reply keyboards/pearlboards/zeus/keymaps/via/keymap.c]: Done sir

File: keyboards/pearlboards/zeus/keymaps/all/keymap.c (hunk 10 above)
Comment (Koobaczech) [reply keyboards/pearlboards/zeus/keymaps/all/keymap.c]: Done sir

File: keyboards/pearlboards/zeus/rules.mk (hunk 11 above)
Comment (Koobaczech) [reply keyboards/pearlboards/zeus/rules.mk]: Done sir

File: keyboards/pearlboards/zeuspad/keymaps/all/keymap.c (hunk 12 above)
Comment (Koobaczech) [reply keyboards/pearlboards/zeuspad/keymaps/all/keymap.c]: Done sir

**Comment by Koobaczech** [comment]
First round of suggestions/changes made :)

**Comment by Koobaczech** [comment]
@drashna Hey sir, can you help me merge this please

File: keyboards/pearlboards/zeuspad/rules.mk [hunk 14]
```diff
@@ -0,0 +1,24 @@
...
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no        # Console for debug
+COMMAND_ENABLE = no        # Commands for debug and configuration
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+NKRO_ENABLE = yes            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes        # Enable keyboard RGB underglow
+AUDIO_ENABLE = no           # Audio output
```
Comment (fauxpark) [inline keyboards/pearlboards/zeuspad/rules.mk:19]:
```suggestion
BOOTMAGIC_ENABLE = no       # Enable Bootmagic Lite
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
NKRO_ENABLE = yes           # Enable N-Key Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
AUDIO_ENABLE = no           # Audio output
```

File: keyboards/pearlboards/zeuspad/readme.md [hunk 15]
```diff
@@ -0,0 +1,15 @@
+# Zeuspad
+
+![Zeuspad](https://i.imgur.com/F657DWw.png)
```
Comment (fauxpark) [inline keyboards/pearlboards/zeuspad/readme.md:3]:
```suggestion
![Zeuspad](https://i.imgur.com/F657DWwh.png)
```

File: keyboards/pearlboards/zeus/rules.mk [hunk 16]
```diff
@@ -0,0 +1,26 @@
...
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no        # Console for debug
+COMMAND_ENABLE = no        # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes        # Enable keyboard RGB underglow
+AUDIO_ENABLE = yes           # Audio output
```
Comment (fauxpark) [inline keyboards/pearlboards/zeus/rules.mk:21]:
```suggestion
BOOTMAGIC_ENABLE = no       # Enable Bootmagic Lite
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
NKRO_ENABLE = yes           # Enable N-Key Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
AUDIO_ENABLE = yes          # Audio output
```

File: keyboards/pearlboards/zeus/readme.md [hunk 17]
```diff
@@ -0,0 +1,15 @@
+# Zeus
+
+![Zeus](https://i.imgur.com/jdCaP1m.jpg)
```
Comment (fauxpark) [inline keyboards/pearlboards/zeus/readme.md:3]:
```suggestion
![Zeus](https://i.imgur.com/jdCaP1mh.jpg)
```

File: keyboards/pearlboards/pearl/rules.mk [hunk 18]
```diff
@@ -0,0 +1,25 @@
...
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no        # Console for debug
+COMMAND_ENABLE = no        # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = yes       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes        # Enable keyboard RGB underglow
+AUDIO_ENABLE = yes           # Audio output
```
Comment (fauxpark) [inline keyboards/pearlboards/pearl/rules.mk:21]:
```suggestion
BOOTMAGIC_ENABLE = no       # Enable Bootmagic Lite
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
NKRO_ENABLE = yes           # Enable N-Key Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
AUDIO_ENABLE = yes          # Audio output
```

File: keyboards/pearlboards/pearl/readme.md [hunk 19]
```diff
@@ -0,0 +1,15 @@
+# Pearl
+
+![Pearl](https://i.imgur.com/xdEtT7u.jpg)
```
Comment (fauxpark) [inline keyboards/pearlboards/pearl/readme.md:3]:
```suggestion
![Pearl](https://i.imgur.com/xdEtT7uh.jpg)
```

File: keyboards/pearlboards/pandora/rules.mk [hunk 20]
```diff
@@ -18,8 +18,7 @@ SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
 NKRO_ENABLE = yes            # USB Nkey Rollover
 BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
 RGBLIGHT_ENABLE = yes        # Enable keyboard RGB underglow
```
Comment (fauxpark) [inline keyboards/pearlboards/pandora/rules.mk:20]:
```suggestion
RGBLIGHT_ENABLE = yes        # Enable keyboard RGB underglow
AUDIO_ENABLE = no            # Audio output
```

File: keyboards/pearlboards/atlas/rules.mk [hunk 21]
```diff
@@ -0,0 +1,26 @@
...
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no        # Console for debug
+COMMAND_ENABLE = no        # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes        # Enable keyboard RGB underglow
+AUDIO_ENABLE = yes           # Audio output
```
Comment (fauxpark) [inline keyboards/pearlboards/atlas/rules.mk:21]:
```suggestion
BOOTMAGIC_ENABLE = no       # Enable Bootmagic Lite
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
NKRO_ENABLE = yes           # Enable N-Key Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
AUDIO_ENABLE = yes          # Audio output
```

File: keyboards/pearlboards/atlas/readme.md [hunk 22]
```diff
@@ -0,0 +1,15 @@
+# Atlas
+
+![Atlas](https://i.imgur.com/jm6DyWA.jpg)
```
Comment (fauxpark) [inline keyboards/pearlboards/atlas/readme.md:3]:
```suggestion
![Atlas](https://i.imgur.com/jm6DyWAh.jpg)
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/pearlboards/atlas/rules.mk [hunk 23]
```diff
@@ -0,0 +1,23 @@
...
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = yes           # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+AUDIO_ENABLE = yes          # Audio output
+ENCODER_ENABLE = yes
+HAPTIC_ENABLE = yes
+HAPTIC_ENABLE = DRV2605L
```
Comment (fauxpark) [inline keyboards/pearlboards/atlas/rules.mk:21]:
```suggestion
HAPTIC_DRIVER = DRV2605L
```

File: keyboards/pearlboards/pandora/rules.mk [hunk 24]
```diff
@@ -18,8 +18,8 @@ SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
...
-AUDIO_ENABLE = no           # Audio output
+AUDIO_ENABLE = no            # Audio output
```
Comment (fauxpark) [inline keyboards/pearlboards/pandora/rules.mk:21]:
```suggestion
AUDIO_ENABLE = no           # Audio output
```

File: keyboards/pearlboards/pearl/rules.mk [hunk 25]
```diff
@@ -0,0 +1,22 @@
...
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = yes           # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+AUDIO_ENABLE = yes          # Audio output
+HAPTIC_ENABLE = yes
+HAPTIC_ENABLE = DRV2605L
```
Comment (fauxpark) [inline keyboards/pearlboards/pearl/rules.mk:20]:
```suggestion
HAPTIC_DRIVER = DRV2605L
```

File: keyboards/pearlboards/zeus/rules.mk [hunk 26]
```diff
@@ -0,0 +1,23 @@
...
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = yes           # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+AUDIO_ENABLE = yes          # Audio output
+ENCODER_ENABLE = yes
+HAPTIC_ENABLE = yes
+HAPTIC_ENABLE = DRV2605L
```
Comment (fauxpark) [inline keyboards/pearlboards/zeus/rules.mk:21]:
```suggestion
HAPTIC_DRIVER = DRV2605L
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

File: keyboards/pearlboards/pearl/readme.md [hunk 27]
```diff
@@ -0,0 +1,15 @@
...
+
+![Pearl](https://i.imgur.com/xdEtT7uh.jpg)
+
+Pearl is a future-proof premium 40%. This is the second round of it, reworked for 2021. This firmware is for the latest QMK PCB's.
+
+* Keyboard Maintainer: Koobaczech
+* Hardware Supported: Pearl
+* Availability: https://pearlboards.net/products/pearl-40
+
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/pearlboards/pearl/readme.md:11]: Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

File: keyboards/pearlboards/zeuspad/readme.md [hunk 28]
```diff
@@ -0,0 +1,15 @@
...
+
+![Zeuspad](https://i.imgur.com/F657DWwh.png)
+
+Zeuspad is the perfect numpad complement to my TKL, Zeus. Standalone it is a glorious numpad with an encoder, oled and rgb.
+
+* Keyboard Maintainer: Koobaczech
+* Hardware Supported: Zeuspad
+* Availability: https://pearlboards.net/products/zeuspadl-coming-2022?variant=39642771718276
+
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/pearlboards/zeuspad/readme.md:11]: Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**